docs/releases/v2020.1.3.rst

```diff
@@ -0,0 +1,62 @@
+Gluon v2020.1.3
+===============
+
+Bugfixes
+--------
+
+- Fixes a bug in musl which can lead to spurious crashes in fastd and other programs, which alternate between single-
```

Comment:
maybe we want to link the github issues for bugfixes like we did in v2020.1.2 release notes?